google/cloud/aiplatform/metadata/resource.py

```diff
@@ -69,23 +74,32 @@ def __init__(
             project=project, location=location, credentials=credentials,
```

Comment:
@sasha-gitg I still used the __init__ method here instead of the empty constructor as the implementation for both looks the same to me. The only field we would need to set after this call would be _gca_resource which we have special handle for it already, so I think we should be ok just use the normal init method. But please let me know if i missed anything.

Reply:
SGTM. I think if we ever expose this to users we will need remove it.